components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodManager.kt

```diff
@@ -271,14 +274,20 @@ class GitpodManager : Disposable {
         val connect = {
             val originalClassLoader = Thread.currentThread().contextClassLoader
             try {
+                val proxies = CommonProxy.getInstance().select(URL(info.gitpodHost))
```

Comment:
@felladrin Should not it be `"https://$gitpodHost/"`?

Reply:
I'll double-check it in a moment. But according to the variable description, `info.gitpodHost` should be the full URL of the host, including the protocol.

Reply:
Confirmed. These are the values when running a workspace on [gitpod.io](https://gitpod.io):
- `info.gitpodHost`: `https://gitpod.io`
- `info.gitpodApi.endpoint`: `wss://gitpod.io/api/v1`